neutromeratio/tautomers.py

```diff
@@ -123,9 +124,18 @@ def from_mol_to_ani_input(mol:Chem.Mol, nr_of_conf:int, in_solvent:bool=False)->
     
     ani_input =  {'ligand_atoms' : ''.join(atom_list), 
             'ligand_coords' : coord_list, 
-            'ligand_topology' : topology }
+            'ligand_topology' : topology,
```

Comment:
would prefer if `ani_input` were replaced with a class, rather than a dictionary -- otherwise many functions need to remember what the dictionary fields are, whether they're mutable, etc.